### PR TITLE
refactor!: use `tui-icon` instead `tui-svg`

### PR DESCRIPTION
**Pull request by mdlufy**
Closes #

### CODE REVIEW

File: projects/addon-table/components/table-pagination/table-pagination.style.less [hunk 1]
```diff
@@ -22,7 +22,9 @@
 
 .t-checkmark {
     min-width: 1.5rem;
-    border-left: 5px solid transparent;
+    margin-inline-start: 5px;
```
Comment (waterplea) [inline projects/addon-table/components/table-pagination/table-pagination.style.less:25]: That's weird, should probably be 0.25rem. I think we should also have `margin-inline-end: -0.125rem` so the icon is equally far from item edges:
image

File: projects/addon-table/components/table/th/th.style.less [hunk 2]
```diff
@@ -124,6 +124,10 @@
     &:hover {
         color: var(--tui-text-01);
     }
+
+    &-icon {
```
Comment (waterplea) [inline projects/addon-table/components/table/th/th.style.less:128]: Let's just rename it `.t-icon`, no need for it to be nested. Strange that we had that class in HTML with no styles applied to it before.

File: projects/core/components/tooltip/tooltip.component.ts [hunk 3]
```diff
@@ -34,6 +34,11 @@ export class TuiTooltipComponent<C = any> extends TuiHintOptionsDirective {
         return this.appearance || '';
     }
 
+    @HostBinding('class._small')
+    protected get small(): boolean {
```
Comment (waterplea) [inline projects/core/components/tooltip/tooltip.component.ts:38]: Let's drop that, we should not depend on icon names, leave it always small for now.

**Review by waterplea — APPROVED** [review]

**Review by waterplea — APPROVED** [review]

File: projects/core/components/tooltip/tooltip.component.ts (hunk 3 above)
Comment (mdlufy) [reply projects/core/components/tooltip/tooltip.component.ts]: droped

File: projects/addon-table/components/table/th/th.style.less (hunk 2 above)
Comment (mdlufy) [reply projects/addon-table/components/table/th/th.style.less]: renamed

**Review by waterplea — APPROVED** [review]

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**